### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by MG-Cloudflow** [MG-Cloudflow/Intune-Toolkit#1] [Ideas]
- Export to CSV
- Script Assignments

**Comment by mainmankkl** [comment]
I like your tool, but it should support the built-in Intune groups "All users" and "All devices"

**Reply by MG-Cloudflow** [reply]
Hey

Thank you for the feedback.
Yes this is on my to do list and hope to be implementing this soon.

kind regards
maxime

**Reply by MG-Cloudflow** [reply]
Hey this is fixed in the latest release

**Comment by xquadens** [comment]
Awesome tool!

It would be great to be able to backup and restore assignments for individual items and have the ability to modify remediation scripts.  Thanks for building this tool, works great for an alpha!

**Reply by MG-Cloudflow** [reply]
Hey

I'm working on that feature ;-)

**Comment by MG-Cloudflow** [comment]
Hey xquadens

I will put this a bit higher on the list ;-)
and thank you for the kind words.

**Comment by opsora-labs** [comment]
Hi,
thanks for your tool. It has been very useful to use with my customers for way faster assignments and to optimize Intune tenants.
Currently I always export each category and add it to the same csv / xlsx to present to the customer and prepare the changes inside the file before implementing it.
It would be great if you could add a button to export all the cateogries into one file directly.

**Reply by MG-Cloudflow** [reply]
Hey

Thank you for the feedback.
I will have a think on how i can achieve this ;-) because i would want it to be so can select the policy's you want and add them to a list. and in the end export that list.

**Comment by Turak64** [comment]
Just discovered this tool via Get Rubix and it looks great! What I would love to see is an easy search field to find which config profile contains a setting. For example, say if you're setting a desktop wallpaper but are unsure which config profiles are applying these settings, have a search field that can quickly find this for you.

**Comment by joemos1550** [comment]
Is it possible to provide some backpu samples (based on Microsoft's best practices)? Something like templates that one can be based on .

**Reply by MG-Cloudflow** [reply]
Hey can you give some more context?

**Comment by joemos1550** [comment]
Hey there, hope you are doing well!

I was thinking that it would be really helpful if you could provide some
backup files as baseline configurations (device configuration polices,
device restrictions, app configuration policies on Microsoft apps, device
filters, dynamic device groups of Windows 10, Windows 11 devices etc).
The endpoint protection section also has some baselines (bitlocker silent
enablement, firewall & attack surface reduction policies etc).

That would be very handy for tenants with no or little configuration in
Intune. They could restore the backups to their tenant as a starting point.
All the configuration and policies could have no assignments, so that they
can assign them to the groups they need.

Στις Παρ 12 Σεπ 2025, 16:57 ο χρήστης Maxime Guillemin | Cloudflow <
***@***.***> έγραψε:

**Reply by opsora-labs** [reply]
You can use the already perfect tool for backup and restore: https://github.com/Micke-K/IntuneManagement

**Comment by joemos1550** [comment]
Thanks, I'll check it out.

Στις Σάβ 13 Σεπ 2025, 04:53 ο χρήστης ConfigForge Labs <
***@***.***> έγραψε:
